src/Game.cs

```diff
@@ -425,27 +425,31 @@ public void Tick()
 			 * modes across multiple devices and platforms.
 			 */
 
+			/* Sleep resolution accuracy is a contentious matter - SDL sets the timer resolution
+			 * to 1 millisecond on Windows, but many Linux systems have a resolution of 4 milliseconds.
+			 * Sleep can be inaccurate, but schedulers tend to wake us up early.
+			 */
+			TimeSpan noSleepThreshold = TimeSpan.FromMilliseconds(4);
```

Comment:
We should look into this and see if we can find a definitive value instead - this could make systems with even lower resolutions unplayable.